### ISSUE TITLE
[9.3.0] Issue happening again "the following content is not available on this app".

### ISSUE THREAD

**Issue by Croodo** [LuanRT/YouTube.js#649]
### Steps to reproduce

  const yt = await ytjs.Innertube.create({ });

  let info = await yt.getBasicInfo(id, "ANDROID");

### Failure Logs

```shell
None
```

### Expected behavior

Getting correct formats url.

### Current behavior

Fall back to a default video. 

### Version

Default

### Anything else?

**ANDROID Client** stoped working properly,  it showing "the following content is not available on this app" again from last 2 days. 

### Checklist

- [X] I am running the latest version.
- [X] I checked the documentation and found no answer.
- [X] I have searched the existing issues and made sure this is not a duplicate.
- [X] I have provided sufficient information.

**Comment by liaolingfen4** [comment]
same issue i meet, has any update?

**Comment by liaolingfen4** [comment]
anyone who can solve the issue?

**Comment by kane50613** [comment]
Is this related? https://support.google.com/youtube/thread/269521462?hl=en

**Comment by LuanRT** [comment]
Yes it is. Also, if anyone still insists in using the android client, using an account bypasses this error.
